python/cudf_polars/cudf_polars/testing/io.py

```diff
@@ -57,6 +57,8 @@ def write(part: pl.DataFrame, file_path: Path) -> None:
                 raise ValueError(f"Unsupported format: {fmt}")
 
     if n_files == 1:
+        if path.is_dir():
```

Comment:
Sign post: This change just ensures that passing a directory is supported too when `n_files=1`.